kolibri/core/assets/src/views/auth-message.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+
+  <div class="auth-message">
+    <h1>
+      {{ header || $tr('loginPrompt') }}
+    </h1>
+    <p>
+      {{ details || defaultDetails }}
+    </p>
+  </div>
+
+</template>
+
+
+<script>
+
+  const userRoles = [
+    'admin',
+    'admin_or_coach',
+    'learner',
+  ];
+
+  module.exports = {
+    props: {
+      authorizedRole: {
+        type: String,
+        validator(role) {
+          return userRoles.includes(role);
+        },
+      },
+      header: { type: String },
+      details: { type: String },
+    },
+    computed: {
+      defaultDetails() {
+        return `${this.$tr('commandStart')} ${this.$tr(this.authorizedRole)} ${this.$tr('commandEnd')}`;
```

Comment:
Better if you pass in the role to the translation to have something like `You must be signed in as {role} to view this page`